.buildkite/_forge.rayci.yml

```diff
@@ -129,6 +129,18 @@ steps:
   - name: rllibbuild
     wanda: ci/docker/rllib.build.wanda.yaml
     depends_on: oss-ci-base_ml
+    env:
+      IMAGE_FROM: cr.ray.io/rayproject/oss-ci-base_ml
+      IMAGE_TO: rllibbuild
+      IS_GPU_BUILD: "false"
```

Comment:
can we change `IS_GPU_BUILD` to `RAY_GPU_BUILD` or `RAYCI_IS_GPU_BUILD` or something? like add a prefix so that it is not very likely to collide with other things?

Reply:
sound good!